### PR TITLE
Dsg 1421/publish audit events through dsgov-events

### PR DESCRIPTION
**Pull request by jgnuvalence**
## Types of changes

Patch
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Documentation change (documentation only, will not require redeployment)
- [ ] Refactor or cleanup (functionality unchanged)
Minor
- [ ] New feature (non-breaking change which adds functionality)
Major
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have performed a self-review of my own code
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules

- [ ] I have confirmed all acceptance criteria defined by the story have been met

### CODE REVIEW

File: settings.gradle [hunk 1]
```diff
@@ -12,6 +12,5 @@ rootProject.name = 'work-manager'
 include 'service'
 include 'client'
 include 'acceptance-tests'
-include 'audit-service-client'
 include 'notification-service-client'
```
Comment (jcsuarezm) [inline settings.gradle:15]: I think this should be deleted too.

File: config/checkstyle/checkstyle.xml [hunk 2]
```diff
@@ -44,7 +44,7 @@
 
     <module name="LineLength">
         <property name="fileExtensions" value="java"/>
-        <property name="max" value="100"/>
+        <property name="max" value="120"/>
```
Comment (jcsuarezm) [inline config/checkstyle/checkstyle.xml:47]: I agree that extending the line length could be a good idea too. I might be wrong but I think Google's style is set to 100, but overall 120 seems like a better fit for our project, but just wanted to point this out.

**Review by jcsuarezm — APPROVED** [review]

File: settings.gradle (hunk 1 above)
Comment (jgnuvalence) [reply settings.gradle:15]: you are right, I'll delete them

File: config/checkstyle/checkstyle.xml (hunk 2 above)
Comment (jgnuvalence) [reply config/checkstyle/checkstyle.xml:47]: I see several parts in our project with lines longer than 100 chars, I think that if we want to modify these things we should create a ticker to fix this in a general way, for now I will leave it like this, do you agree?

**Review by jcsuarezm — APPROVED** [review]

**Pull request merged**